### ISSUE TITLE
Site is down (https://guide.elm-lang.org/)

### ISSUE THREAD

**Issue by bowsersenior** [evancz/guide.elm-lang.org#201]
The https://guide.elm-lang.org/ is down. Looks like it is an issue with GitBook. This started about 2 hours ago. I was able to use the site fine earlier today.

See this screenshot:

<img width="1401" alt="Screen Shot 2019-06-19 at 5 42 04 PM" src="https://user-images.githubusercontent.com/40264/59803615-019ab800-92ba-11e9-989d-e6ca0f6a8bed.png">

Also when I try to look at the GitBook page at https://legacy.gitbook.com/book/evancz/an-introduction-to-elm/details, I get a 500 page:

<img width="1093" alt="Screen Shot 2019-06-19 at 5 43 29 PM" src="https://user-images.githubusercontent.com/40264/59803641-12e3c480-92ba-11e9-85ea-0a231ee63867.png">

**Comment by ssimontis** [comment]
Confirming that this is an issue as well, although last time I accessed successfully was yesterday.

**Comment by evancz** [comment]
When I first started using GitBook, they would host with a custom domain. It looks like this publishing path has become "legacy" and may not continue to exist. According to their [README](https://github.com/GitbookIO/gitbook):

> Content hosted on the legacy.gitbook.com will continue working until further notice.

So I switched the hosting over to netlify. The site is back up for me, and it should come back as the new DNS information propagates to your region.
